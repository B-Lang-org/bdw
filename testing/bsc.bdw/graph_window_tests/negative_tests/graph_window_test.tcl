
# Opens the specified graph window
handlers::show_graph 

# Opens the specified project
handlers::open_project project.bspec

# Opens the specified graph window
handlers::show_graph 

# Opens the specified graph window
handlers::show_graph -con 

# Opens the specified graph window
handlers::show_graph -combined_full 

# Closes the project 
handlers::close_project

# Exits the workstation
exit
