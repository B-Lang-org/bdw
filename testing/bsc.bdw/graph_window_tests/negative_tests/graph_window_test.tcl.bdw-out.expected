There is no open project
The window is not specified.
Unknown option: -con
There is no .dot file for the currently  loaded module
